src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -798,6 +792,28 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  private def entityLessThan(attributeName: String)(e1: Entity, e2: Entity): Boolean = {
```

Comment:
the following changes to the test are because rich man's sorting is a little different: nulls come first not last and types are sorted separately